client/package/_colors.scss

```diff
@@ -0,0 +1,13 @@
+//TODO: use v2 palette
```

Comment:
It’d be safer to add a packagey prefix/suffix to these colours—variables aren’t file or project scoped, so anything consuming this component could get its variables redefined